aioftp/client.py

```diff
@@ -428,39 +409,11 @@ def parse_ls_date(s):
 
         :rtype: :py:class:`str`
         """
-        if not (s[3] == ' ' and s[6] == ' '):
-
-            raise ValueError
-
-        month = "{0:02d}".format(time.strptime(s[:3], "%b").tm_mon)
-        day = s[4:6]
-
-        if day[0] == ' ':
-
-            day = '0' + day[1:]
-
-        if s[9] == ':':
-
-            year = "0000"
-            hour_min = s[7:9] + s[10:12]
-
-        else:
-
-            hour_min = "0000"
-
-            if s[6] != ' ':
-
-                raise ValueError
-
-            year = s[8:12]
-
-        result = year + month + day + hour_min
-
-        if not result.isdigit():
-
-            raise ValueError
-
-        return result + "00"
+        try:
+            d = time.strptime(s, "%b %d %H:%M")
+        except ValueError:
+            d = time.strptime(s, "%b %d  %Y")
+        return time.strftime("%Y%m%d%H%M00", d)
```

Comment:
We need english locale for non-english users. So, use example from link from my comment https://github.com/pohmelie/aioftp/pull/50#issuecomment-284867736
This context manager should be placed in common, since it will be used in server too.